src/tests/testData.js

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const merge = require('lodash.merge');
+
+const carousel1 = {
+  component: {
+    carousel: {
+      carousel1: {
+        id: '/content/mysite/en/home/jcr:content/root/carousel1',
+        items: {}
+      }
+    }
+  }
+};
+
+const image1 = {
```

Comment:
is it needed as there is a image1 in the test data below?

Reply:
Same as for carousel content